pyspark_ai/prompt.py

```diff
@@ -41,20 +41,21 @@
 )
 
 SPARK_SQL_EXAMPLES = [
-    """QUESTION: Given a Spark temp view `spark_ai_temp_view_14kjd0` with the following columns:
+    """QUESTION: Given a Spark temp view `spark_ai_temp_view_14kjd0` with the following sample vals,
```

Comment:
nit: we can have a follow-up to create a function to unify the prompt over a table. So that both the example and the prompt for user input are consistent.